api/app/models/player.rb

```diff
@@ -1,3 +1,7 @@
 class Player < ApplicationRecord
+  PREFERENCES = %w[no_preference attacker defender]
+
+  enum preference: PREFERENCES
```

Comment:
We probably should consider dropping the `PREFERENCES` constant, since when defined as `enum` it will be available through `Player.preferences`. What do you think?

Reply:
I added constant to avoid calling Hash#keys, but as I will replace int with string representation it will be removed.